File: mgyoutube-multitech/api-go/src/app/youtubeproperties.go
package main

import (
	"encoding/json"
	"io/ioutil"
)

// YouTubeProperties - YouTube properties read from a config file
type YouTubeProperties struct {
	APIKey          string `json:"apikey"`
	ApplicationName string `json:"applicationame"`
}

// ReadYouTubeProperties - read the config file and return the YouTube properties
func ReadYouTubeProperties() (*YouTubeProperties, error) {

	youTubeProperties := YouTubeProperties{}

	file, errReadFile := ioutil.ReadFile("secrets/youtube.json")
	if errReadFile != nil {
		return nil, errReadFile
	}

	errUnmarshal := json.Unmarshal([]byte(file), &youTubeProperties)

	if errUnmarshal != nil {
		return nil, errUnmarshal
	}

	return &youTubeProperties, nil
}

File: mgyoutube-multitech/api-node/src/youtubeproperties.ts
import { readFileSync } from "fs"

export class YouTubeProperties {
    public apiKey: string;
    public applicationName: string;

    constructor() {
        var json = readFileSync("secrets/youtube.json");
        var jsonObject = JSON.parse(json.toString());

        this.apiKey = jsonObject.apiKey;
        this.applicationName = jsonObject.applicationName
    }
}